dev/integration_tests/wide_gamut_test/integration_test/app_test.dart

```diff
@@ -79,6 +92,8 @@ bool _findBGRA10Color(
   expect(bytes.lengthInBytes, width * height * 8);
   expect(bytes.lengthInBytes, byteData.lengthInBytes);
   var foundDeepRed = false;
```

Comment:
This function looks like it got updated incorrectly. Make sure to update it to match the others.